File: test/fixtures/python/clean.py
# Test fixture: clean file with no secrets
import os
import openai

api_key = os.getenv("OPENAI_API_KEY")
openai_client = openai.OpenAI(api_key=api_key)

message = "Hello, world!"
name = "sk-learn-model-v1"
description = "This is just a normal string"

File: test/fixtures/javascript/clean.js
// Test fixture: clean JavaScript file with no secrets
const apiKey = process.env.OPENAI_API_KEY;

const message = "Hello, world!";
const modelName = "sk-learn-model-v1";

const config = {
  endpoint: "https://api.openai.com/v1",
  timeout: 30000
};
